test/data/Cardano/Wallet/Api/ApiT (Hash "Tx").json

```diff
@@ -0,0 +1,9 @@
+{
+    "seed": -3120370980410250458,
+    "samples": [
+        "c47e3b3120f81d2546795d7d78d45d2b91ad1fea26073e4a477e2c5317410c244f082b32560863084f3c1f716bed1e3d5c854758ba2a1f696d399f388f18466b",
+        "bd1b1817695b42134ff5cc7a023972ca5c3d5c500b235c1a5d0a324f7d3c49284d6a1e51289e5031501f394a283720373c490d414f424d45178c4a27412e763d",
+        "2c47c30f62146920cc32468168daa03f1b127c6f7140044915de1a25541819047f4444224b5408690b480f056141794b38554c736474133e0083461e12787200",
+        "160830005e014910157d434c322f60101e5d785400550e094e7a3e668115782060225c234b7b7f5f3c2b4d345c554568416b4d732a2b4461155c3c4a8d05433b"
+    ]
```

Comment:
Those looks surprisingly long. They are 128-char long :thinking: ?

Reply:
hm - that looks strange because its generated with arbitrary 32 bytes/chars so that should give 64 chars string: https://github.com/input-output-hk/cardano-wallet/blob/dfa7167225ff44477c25333e1613f86e46286f15/test/unit/Cardano/Wallet/Api/TypesSpec.hs#L455 . 

```haskell
> hash <- generate (arbitrary :: Gen (Hash "Tx"))
> hash
Hash {getHash = "\NAKtc@\249X\DELd\DC3(5W\NUL\\]J('\DC2,_@?s\SO<\vq\RS,M\164"}
> encode $ ApiT hash
"\"15746340f9587f6413283557005c5d4a2827122c5f403f730e3c0b711e2c4da4\""
> length "15746340f9587f6413283557005c5d4a2827122c5f403f730e3c0b711e2c4da4"
64
```

Reply:
Aha, I see you have fixed it. Sorry forgot to push regenerated golden tests